scriptworker/worker.py

```diff
@@ -38,43 +37,38 @@
         int: status
     """
     loop = asyncio.get_event_loop()
-    await update_poll_task_urls(
-        context, context.queue.pollTaskUrls,
-        args=(context.config['provisioner_id'], context.config['worker_type']),
-    )
-    for poll_url, delete_url in get_azure_urls(context):
+    try:
+        tasks = await claim_work(context)
+    except ScriptWorkerException:
+        await asyncio.sleep(context.config['poll_interval'])
+        return
+    if tasks and tasks.get('tasks', []):
+        log.info(tasks)
+        context.claim_task = tasks['tasks'][0]  # scriptworker can only handle 1 concurrent task
```

Comment:
We really shouldn't be logging the task, which includes temp cred info ... so thank you for reminding me to take my debug logging out :D